x/stake/endblock.go

```diff
@@ -64,6 +76,10 @@ func EndBreatheBlock(ctx sdk.Context, k keeper.Keeper) (validatorUpdates []abci.
 			publishCompletedUBD(k, completedUbds, sideChainIds[i], ctx.BlockHeight())
 			publishCompletedRED(k, completedREDs, sideChainIds[i])
 		}
+		if sdk.IsUpgrade(sdk.BEPHHH) {
+			// distribute beacon chain rewards
+			k.DistributeInBreathBlock(ctx, types.MockSideChainIDForBeaconChain)
```

Comment:
I think it is better to move `DistributeInBreathBlock` up, just after `storeValidatorsWithHeight`.

Reply:
The bscToBcFee is collected in the distribution of BSC rewards, so Beacon Chain distribution should be after BSC distribution.

Reply:
I c.